test/README.md

```diff
@@ -0,0 +1,95 @@
+## How To Test Verdaccio
+
+Welcome to the testing folder at Verdaccio. This document aims to help you to go throug how Verdaccio show be tested.
+
+First of all we must explain the stack, we use `jest` and other tools as `supertest` to be able test the API and `puppetter` for End to End testing.
+
+We go along with the following rules in order to be consisten with all test and will make your code review smooth and fast:
```

Comment:
consisten => consistent
all test => all tests
and will make => which will make